Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM debian:6
+FROM ubuntu:trusty
 MAINTAINER Bilge <bilge@scriptfusion.com>
 
 WORKDIR	/root
 
 # Install dependencies.
 RUN	apt-get update && DEBIAN_FRONTEND=noninteractive\
-	apt-get install -y build-essential gperf bison flex texinfo wget gawk libtool automake libncurses5-dev
+	apt-get install -y build-essential gperf bison flex texinfo wget gawk libtool automake libncurses5-dev help2man
```

Comment:
It was a dependency of crosstool 1.22 - we didn´t investigate further on that, we just added it.